src/host/ut_host/ScreenBufferTests.cpp

```diff
@@ -4346,6 +4347,117 @@ void ScreenBufferTests::DeleteChars()
     }
 }
 
+void ScreenBufferTests::HorizontalScrollOperations()
+{
+    BEGIN_TEST_METHOD_PROPERTIES()
+        TEST_METHOD_PROPERTY(L"Data:op", L"{0, 1, 2, 3}")

```

Comment:
Instead of adding cleanup annotations, you can also use method isolation. It's maybe not worth changing _now_, but worth considering in the future :)

I don't know what downsides exist... so that could be a problem.

Reply:
It depends how much cleanup is involved, but I usually try to avoid method isolation if I can help it, because I thought it made the tests quite a bit slower. It also makes them more painful to debug, because you can only step through a single iteration and then it complains about TAEF needing to start a second test host (I don't know if there's a workaround for that which I'm just not aware of).